source/guide_fittrackee.rst

```diff
@@ -0,0 +1,194 @@
+.. highlight:: console
+.. author:: Astrid Günther <https://astrid-guenther.de>
+
+.. tag:: lang-python
+.. tag:: fitness-tracker
+.. tag:: sport-tracking-app
+.. tag:: tacking
+.. tag:: tacker
+.. tag:: sport
+.. tag:: fitness 
+.. tag:: gpx   
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/fittrackee.png
+      :align: center
+
+##########
+Fittrackee
+##########
+
+.. tag_list::
+
+Fittrackee_ allows you to track your outdoor activities (workouts) from gpx files and keep your data on your own server. 
+It is a open source and decentralized sport tracking app. 
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Python <lang-python>` and its package manager pip
+  * :lab:`PostgreSQL <guide_postgresql>`
+  * :manual:`Firewall Ports <basics-ports>`
+  * :manual:`Web Backends <web-backends>`
+  * :manual:`Supervisord <daemons-supervisord>`
+  * :lab:`Redis <guide_redis>`
+  * :manual:`Domains (optional) <web-domains>`
+  * :manual:`Mail (optional) <mail-access>`
+
+
+License
+=======
+
+Fittrackee is open-sourced software licensed under the `AGPL license <https://github.com/SamR1/FitTrackee/blob/master/LICENSE>`_.
+
+Prerequisites
+=============
+
+Mandatory `Prerequisites for Fittrackee <https://samr1.github.io/FitTrackee/en/installation.html#prerequisites>`_:
+
+  * :manual:`Python <lang-python>` in the version >3.8.1. 
+  * :manual:`PostgreSQL <guide_postgresql>` in the version 11+. 
+
+Installation
+============
+
+Create folder
+-------------
+
+We do not install Fittrackee in the :manual:`DocumentRoot <web-documentroot>`. We install it in a separate folder. First we create this folder.
```

Comment:
No, we don't install it in a separate folder, please check out the [STYLEGUIDE](https://github.com/Uberspace/lab/blob/main/STYLE.md):
> Don't mention additional document roots. Keep it simple. Don't use subfolders. Always use the standard document root ~/html. Always assume the document root is empty.